hi-ml/src/health_ml/model_trainer.py

```diff
@@ -63,6 +65,8 @@ def create_lightning_trainer(container: LightningContainer,
     :param container: The container with model and data.
     :param resume_from_checkpoint: If provided, training resumes from this checkpoint point.
     :param num_nodes: The number of nodes to use in distributed training.
+    :param azureml_run_for_logging: The AzureML Run object to which all metrics should be logged. If None and the
```

Comment:
Why would we want to log to any run other than the current one?

Reply:
Think I understand the scenario -  the code is running outside of AML so we create a run to log to?

Reply:
clarified.